src/_posts/databases/postgresql/2000-01-01-backing-up.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Backing Up Your Scalingo for PostgreSQL® Addon
+nav: Backing Up
+modified_at: 2023-12-06 00:00:00
+tags: databases postgresql addon
+index: 7
+---
+
+
+Starter and Business Plans of Scalingo for PostgreSQL® include automated and
+managed backups so you don't have to worry about them.
+
+We use 2 kinds of mechanisms to create these automated backups:
+[Point-in-Time Recovery](#understanding-point-in-time-recovery) backups and
+[On-Demand](#understanding-on-demand-backups) backups. [Manual backups](#understanding-manual-backups)
+are also available for these Plans.
+
+
+## Understanding Point-in-Time Recovery Backups
+
+Point-in-time recovery (PITR) allows you to ask for the restoration of your
+data at a specific date. We achieve this by making a full PITR backup of the
+database weekly. Between two full PITR backups, we keep track of the
+write-ahead logs (WAL). The WAL contains all the modification instructions. By
+replaying the WAL from a full PITR backup to a specific date, we are able to
+rebuild the state of the database at that particular date.
+
+You have nothing to do to be able to use the PITR mechanism. Be aware that you
+can only use the PITR on the period between now and now minus seven days.
+
+### Retention Policy for PITR Backups
+
+| Plan     | PITR Backup Retained |
+| -------- + -------------------- |
+| Sandbox  | N/A                  |
+| Starter  | 1                    |
+| Business | 1                    |
+
+
+## Understanding On-Demand Backups
+
+On-Demand backups are done on a daily basis. They consist in dumping your
+database in an archive that we keep (secured) during a certain amount of time.
```

Comment:
issue: I don't really understand the word "secured" between parenthesis. Either it's important to mention and it shouldn't be between parenthesis. In that case I would explain what does "secure" means. Or this information is not that important and it's OK to just drop it. 

I would drop it.

```suggestion
database in an archive that we keep during a certain amount of time.
```